File: src/main/java/com/example/backend/repository/TarotCardRepository.java
package com.example.backend.repository;

import com.example.backend.documents.TarotCard;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

import java.util.UUID;

@Repository
public interface TarotCardRepository extends MongoRepository<TarotCard, UUID>
{

}

File: src/main/kotlin/com/example/backend/repos/TarotCardRepository.kt
package com.example.backend.repos

import com.example.backend.documents.TarotCard
import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface TarotCardRepository : ReactiveMongoRepository<TarotCard, UUID> {

}